package.json

```diff
@@ -4,7 +4,7 @@
   "license": "Apache-2.0",
   "author": "SAP, Spartacus team",
   "engines": {
-    "node": "^16.13.0 || ^18.10.0 || ^20.10.0"
+    "node": "^18.13.0 || ^20.10.0"
```

Comment:
Would you mind running `npm i` locally? I believe then `package-lock.json` will be updated as well. If yes, then please commit it as well.